vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -776,6 +780,11 @@ testPkgDownloaded() {
       if [ -z "${rpmFile}" ]; then
         err $test "Package ${packageName}-${packageVersion} does not exist, content of downloads dir is $(ls -al ${downloadLocation})"
       fi
+    elif [ $OS = $MARINER_OS_NAME ] || { [ $OS = $AZURELINUX_OS_NAME ] && [ $OS_VERSION = "2.0" ]; }; then
```

Comment:
We don't have package for Az Linux 2.0. This should've failed for that Az Linux 2.0.

Reply:
right, confused as well why this if clause is needed